src/css/flags/overlay.less

```diff
@@ -5,4 +5,9 @@
     .jw-controls-right .jw-logo {
         display: none;
     }
+    &-sharing, &-related {
```

Comment:
I think we only have sharing and related overlays, so we do not need to specify &-sharing, &-related